docs/validation.md

```diff
@@ -0,0 +1,54 @@
+# Validation helpers
+
+Validation system is often ignored with Symfony, no time for it, "not required for v0", esthetic feature...
+But users make mistakes. All the time. Errors have to be consider as an application use case.
+
+These helpers dont add killer features to framework, just developper experience improvements.
+
+## Yaml file parsing
+
+Maybe you red it before, but for us, annotations are wrong. Link a class and her configurations breaks everything we believe in, when we use dependency injection elsewhere.
```

Comment:
you red -> you read (même si c'est du passé)

annotations are wrong ->
annotations are bad

Link a class and here configurations ->
Link a class and its configurations